tinygrad/ops/ops_gpu.py

```diff
@@ -2,9 +2,16 @@
 import numpy as np
 from tinygrad.helpers import binary_broadcast
 from ..tensor import Function
-from ..llops.gpu import GPUBuffer
-from ..llops.gpu import unary_op, binary_op, reduce_op, perm_axis, inner_slice
-from ..llops.gpu import matmul, conv, convdw, convdx
+
+import os
```

Comment:
Is it something in the gist of this you have in mind? The goal is to only use ops_gpu and get rid ops_cuda file? @geohot